diskplorer.py

```diff
@@ -121,7 +123,7 @@ def global_section():
             runtime={args.test_step_time_seconds}s
             time_based=1
             ramp_time=5
-            filename={args.device}
+            filename={args.filename}
```

Comment:
The test is designed to work on the raw device and not require the user to create a file (how will they know the size?)

Reply:
`filename`'s size could be specific by `size-limit`
eg.

```bash
 sudo ./diskplorer.py /dev/nvme1n1 --filename=/data/tmp/tempfile --result-file koxia.json --size-limit=1G
```

Reply:
Well, it makes the whole thing harder to use, since now the user has to prepare the system.

Reply:
if user dont specific `args.filename`, `args.filename` equal with `args.device`

change diskexploer's behavier like that?

Reply:
Well, maybe, but I wasn't able to guess it, and neither will the user.